go-controller/pkg/ovn/pods.go

```diff
@@ -159,44 +159,43 @@ func (oc *Controller) deleteLogicalPort(pod *kapi.Pod) {
 		annotation, err := util.UnmarshalPodAnnotation(pod.Annotations)
 		if err == nil {
 			podIfAddrs := annotation.IPs
-			_ = oc.lsManager.ReleaseIPs(pod.Spec.NodeName, podIfAddrs)
+			err = oc.lsManager.ReleaseIPs(pod.Spec.NodeName, podIfAddrs)
```

Comment:
this will error if the node is not in logical switch manager cache. We need to reconcile a pod retry and the node being gone. We should check here upon error if the node exists anymore in kapi. If it doesn't then we can ignore the release IP error. Or maybe we should do that in the ReleaseIPs function.